pjmedia/src/pjmedia/av_sync.c

```diff
@@ -330,11 +326,15 @@ PJ_DEF(pj_int32_t) pjmedia_av_sync_update_pts(
         pj_sub_timestamp(&ntp_diff, &media->last_ntp);
         ms_diff = ntp_to_ms(&ntp_diff);
 
+        /* Smoothen and round down the delay */
+        ms_diff = ((ms_diff + 19 * media->smooth_diff) / 200) * 10;
```

Comment:
Should an upper bound considered? Where does the magic numbers come from?

Reply:
Just added the upper bound 60s in the last commit, actually not really sure if it is needed.

The magic numbers are weighting for the smoothing, i.e: last delay uses weight of 19, newly received delay uses weight of 1, then divide the sum by 20 for the new delay. Instead of by 20, it is by 200 then multiplied to 10 for rounding down to nearest 10.

Thanks for the feedback.